src/Symfony/Component/ObjectMapper/Tests/ObjectMapperTest.php

```diff
@@ -235,13 +235,12 @@ public function __get(string $key): string
         $this->assertSame('test', $mapped->mappedName);
     }
 
-
-    public function testTransformToWrongValueType(): void
+    public function testTransformToWrongValueType()
     {
         $this->expectException(MappingTransformException::class);
         $this->expectExceptionMessage('Cannot map "stdClass" to a non-object target of type "string".');
 
-        $u = new \stdClass;
+        $u = new \stdClass();
```

Comment:
Shall we enable:
https://cs.symfony.com/doc/rules/operator/new_with_parentheses.html#example-1